tests/static-api/_expected/v1/repos.json

```diff
@@ -16,11 +16,15 @@
       "branch_protections": [
         {
           "pattern": "master",
-          "ci_checks": [
-            "CI"
-          ],
           "dismiss_stale_review": false,
-          "required_approvals": 1,
+          "mode": {
+            "PrRequired": {
+              "ci_checks": [
+                "CI"
+              ],
+              "required_approvals": 1
+            }
+          },
```

Comment:
this seems like a breaking change to the API. idk if anything is using the existing keys, but since some of them are removed I guess it should treated as potentially breaking?

Reply:
https://github.com/rust-lang/sync-team/pull/75 will also be needed to be merged first, since it adds support for `RenovateBot` (`sync-team` now doesn't compile after updating `team` since the bot was added).